reference/content-types/text_editor.rst

```diff
@@ -22,6 +22,21 @@ Parameters
     * - links
       - boolean
       - Adds buttons for creating links to the text editor
+    * - scripts
+      - boolean
+      - Allow script tags in source code
+    * - iframes
+      - boolean
+      - Allow iframe tags in source code
+    * - height
+      - integer
+      - Set the text editor height
+    * - maxHeight
+      - integer
+      - Set the max height for autogrow '0' is  infinitely
+    * - enterMode
+      - string P/BR/DIV
```

Comment:
I think this should simply be `enum`. The values should be given in the description.